_commands/basics/gzip.md

```diff
@@ -0,0 +1,24 @@
+---
+---
+
+gzip
+-------
+
+Gzip reduces the size of the name files by compressing ( or "packaging") a file into an "archive file". It will only attempt to compress regular files. Compressed files can be restored to their original form using gzip -d or gunzip or zcat.
```

Comment:
Careful not to confuse people here, `tar` is an archive-creating utility, `gzip` is a compression utility. Technically gzip _can_ create archives, but it's almost never done (unlike `zip`, which is both a compression and archive format). I wouldn't use the word archive here.